src/misc/article.v3.yaml

```diff
@@ -0,0 +1,85 @@
+$schema: http://json-schema.org/draft-04/schema#
```

Comment:
This file looks pretty much the same as the `src/misc/article.v2.yaml` file, is it necessary to add this new one, or maybe I don't understand fully?

Reply:
There's one difference and that is the reference to `- $ref: ../snippets/article.v2.yaml` rather than `- $ref: ../snippets/article.v1.yaml`. I believe it may be possible to inherit from `src/misc/article.v2.yaml` and just articulate the changes from that file but I can't find another example of that approach in this project. Although that approach may make the code reviews easier it may make it more difficult to know the current status of the schema as each version just contains a delta.

Reply:
Thanks, it sounds like technically possible but it's different for stylistic reasons, sounds fine to me, and it clarifies things for me.

Reply:
Oh, I think I see my error, and I see the necessity to bump it now